From: Josselin Mouette <joss@debian.org>
Date: Tue, 19 Dec 2017 18:31:53 -0500
Subject: Revert ABI breakage

Provide the rsvg_handle_new_gz function.
Do not provide the C prototype to force
applications using it to use rsvg_handle_new instead.
---
 rsvg-gobject.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/rsvg-gobject.c b/rsvg-gobject.c
index 2a4726c..423069d 100644
--- a/rsvg-gobject.c
+++ b/rsvg-gobject.c
@@ -380,3 +380,9 @@ rsvg_handle_new_with_flags (RsvgHandleFlags flags)
                          "flags", flags,
                          NULL);
 }
+
+RsvgHandle *
+rsvg_handle_new_gz (void)
+{
+    return rsvg_handle_new ();
+}
